### PR TITLE
Added documentation for the contentBodyStringBuilder method of AMQCom…

### PR DESCRIPTION
**Pull request by thearashs**
Minor change in AMQCommand.java:
- Added method documentation for contentBodyStringBuilder in AMQCommand,java

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@asetayeshi Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-java-client&pullRequestId=656)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-java-client&pullRequestId=656) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@asetayeshi Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-java-client&pullRequestId=656)!

File: src/main/java/com/rabbitmq/client/impl/AMQCommand.java [hunk 1]
```diff
@@ -152,6 +152,12 @@ public String toString(boolean suppressBody){
         }
     }
 
+    /**
+     * Instantiates and returns a StringBuilder
+     * @param body  Array of bytes that represents the content body
+     * @param suppressBody  A boolean that demonstrates if content body should be ignored
```
Comment (michaelklishin) [inline src/main/java/com/rabbitmq/client/impl/AMQCommand.java:158]: This raises a question: what would be returned if the body is ignored. The answer is its length. If we are to document this method at all (not a part of the public API, hardly of much interest to most users and even contributors), we should make it reflect that.

**Comment by michaelklishin** [comment]
Thank you for taking a moment to contribute! I think we should clarify the `suppressBody` argument.

**Pull request closed without merging**